frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -1132,6 +1142,23 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
                               },
                           }
                         : null,
+                    featureFlags[FEATURE_FLAGS.ERROR_TRACKING]
```

Comment:
I just used the error tracking FF, given we'll probably release this when we release that